pyvcloud/vcd/org.py

```diff
@@ -77,6 +77,32 @@ def create_catalog(self, name, description):
             EntityType.ADMIN_CATALOG.value,
             catalog)
 
+    def create_role(self, role_name, description, rights):
+        """
+        Creates a role in the organization
+        :param role_name: name of the role to be created
```

Comment:
Let's also specify the type of each argument. 
like 
:param role_name:  (str): name of the role to be created